x-pack/plugins/infra/public/components/asset_details/tabs/overview/metrics/metrics_section.tsx

```diff
@@ -107,13 +107,14 @@ export const MetricsSectionCompact = ({
 const Section = ({
   title,
   dependsOn = [],
+  shouldCollapse = false,
```

Comment:
I think you could probably name this: 

```suggestion
  collapsible = false,
```